services/eth.json

```diff
@@ -11,8 +11,9 @@
         {
             "name": "free",
             "free": true,
-            "image": "nimak/eth-dev",
-            "description": "Free Trial"
+            "image": "docker.io/nimak/eth-dev",
```

Comment:
This does not solve the problem.
When we pull it does use docker.io. We don't actually know if it successfully pulled the image or not.
But we find another problem when creating a container it starts looking for the image "docker.io/nimak/eth-dev" which is incorrect. So we shouldn't add the `docker.io` to the image name. I think for the time being hard coding it makes the most sense and then if we start pulling images from outside of docker hub then we can add a repo field in the config.